README.md

```diff
@@ -21,3 +21,10 @@ jekyll serve
 ```
 
 View the local site at http://localhost:4000.
+
+### Resizing icons
+
+To resize all the PWA icons in images/:
```

Comment:
Should "images/" have backticks around it? Feels weird to have the slash floating here?

Reply:
✅